app/src/main/java/com/example/p1/ui/theme/WelcomePage.kt

```diff
@@ -0,0 +1,145 @@
+package com.example.ttrip.ui.theme
+import androidx.compose.foundation.BorderStroke
```

Comment:
Package directive does not match the file location, Welcome Screen shouldn't live in the theme module,
Create a folder for the features and split it by feature for now :D 

<img width="1238" alt="Screen Shot 2024-11-03 at 10 34 34 AM" src="https://github.com/user-attachments/assets/cb39990f-b48e-45aa-965d-4f34ae0facff">